Add announcement blog post with screenshots (#107)
statelyai/blog: content/posts/2022-10-18-stately-studio-1-0.mdx

---
title: Introducing Stately Studio 1.0
description: We’re excited to announce the release of Stately Studio 1.0! We’ve been working on Stately Studio for over a year now, and we’re thrilled to share it with you.
tags:
  - editor
  - studio
  - announcement
author: David Khourshid
ogImage: "https://stately.ai/blog/2022-10-18-introducing-stately-studio-1-0.png"
publishedAt: "2022-10-18"
---

> We’re excited to announce the release of Stately Studio 1.0! 🚀 We strongly believe that app logic should be visually collaborative, and we’ve been worked hard to make that a reality. We’ve been working on Stately Studio for over a year now, and we’re thrilled to share it with you.

Reviewer: ```suggestion
We’re excited to announce the release of Stately Studio 1.0! 🚀 Our mission is for all app logic should be visually collaborative and accessible to your entire team, and we’ve been working hard to make that a reality.
```

Private/unlisted machines and team roles are available on [our new Pro subscription.](https://stately.ai/pricing)

## Editor redesign

You might have noticed a refreshed redesign of the editor. The editor view is now more project-centric, allowing you to switch between machines with a single click. And we’ve improved the layout of states to give you more space to create and rearrange states. 

![Simulate view in the Stately Studio showing a video player machine. In the mini state of the machine, there’s an image of a video player embedded in the description. There’s a popover menu showing the simulation event log. There’s a drawer icon in the top left showing where the tree view is hidden away. A breadcrumb in the header shows this machine is in the Stately team and Learn Stately studio project. The current tab in the top navigation is Simulate, but Edit is also available.](studio-1-0-simulate.png)

> We’ve also redesigned guarded transitions. They are now individual transitions connected with a dotted line and a diamond, which should feel natural to anyone that has worked with flowcharts. The separated transitions make it easier to read and arrange the conditions for each transition, allowing you to change the source and target states of individual transitions quickly.

Reviewer 2: A screenshot of new guarded transitions design would be epic.

Simulation mode has also been redesigned, making it easier to see active states and transitions. There are many more improvements to the editor, but we’ll leave them to you to discover.

## More export options

You asked, and we listened (we’re currently in the listening state). We’ve added more export options beyond JSON, including JavaScript and TypeScript. We have plans to add more export options in the future, including:

- Markdown
- CodeSandbox
- StackBlitz
- SVG, PNG, PDF
- And others 

[Request your favorite export options on our Canny roadmap](https://statelyai.canny.io).

![The code popover panel showing over a video player machine. The Code options are JSON, JavaScript and TypeScript. The JavaScript option is selected, alongside a button to copy to clipboard. The code is shown in a text area below as JavaScript object describing the video player machine.](studio-1-0-export.png)

## What’s next?

We have so many things planned for the future:

- A Stately GitHub integration for public and private repos

> - XState parity: supporting all of XState’s features in the Stately Studio (we’re almost there!)

Reviewer 2: Link to XState here?

- More export options and an embeddable editor
- Support for other graph-based tools, CI/CD pipelines and other workflow engines/DSLs
- Static graph analysis to help you catch logic errors ahead of time, which is difficult to do with plain code
- Many more examples, tutorials, videos, and documentation

## Try out the Studio

> All our new Pro features are available with a 30 day free trial you can try with your whole team. Upgrade today using the Upgrade button in [the Stately Studio](https://stately.ai/editor) or Sign in with GitHub to create a new account.

Reviewer 2: I don't know how to emphasize more on the "30 days free trial" here.

> All our new Pro features are available with a 30 day free trial you can try with your whole team. Upgrade today using the Upgrade button in [the Stately Studio](https://stately.ai/editor) or Sign in with GitHub to create a new account.

Reviewer 2: Screenshot of the upgrade button? Unsure users can find it easily.
